lib/XMLHttpRequest.js

```diff
@@ -328,12 +329,14 @@ function XMLHttpRequest(opts) {
           } else {
             self.status = 200;
             self.responseText = data;
+            self.response = Buffer.from(fs.readFileSync(unescape(url.pathname), 'binary'), 'binary');
             setState(self.DONE);
           }
         });
       } else {
         try {
           this.responseText = fs.readFileSync(unescape(url.pathname), 'utf8');
+          this.response = Buffer.from(fs.readFileSync(unescape(url.pathname), 'binary'), 'binary');
```

Comment:
You're now effectively duplicating the work. I'd prefer a way where we can use the result of one `readFileSync` to provide both `responseText` and `response` with data.